driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/unified/UnifiedReactiveStreamsTest.java

```diff
@@ -50,4 +54,9 @@ protected MongoClient createMongoClient(final MongoClientSettings settings) {
     protected GridFSBucket createGridFSBucket(final MongoDatabase database) {
         return new SyncGridFSBucket(GridFSBuckets.create(((SyncMongoDatabase) database).getWrapped()));
     }
+
+    @Override
+    protected ClientEncryption createClientEncryption(final MongoClient mongoClient, final ClientEncryptionSettings clientEncryptionSettings) {
+        return new SyncClientEncryption(new ClientEncryptionImpl(((SyncMongoClient) mongoClient).getWrapped(), clientEncryptionSettings));
```

Comment:
No - passing in a MongoClient is only available via the non public API

Reply:
It can be added to the public API but feel that would be a different ticket.

Reply:
Yes, definitely a different ticket.  It was left out intentionally, so we would need to evaluate if it's a good idea to add it.  Taking just MongoClientSettings allows the ClientEncryptions to manage the lifetime of the key vault client (note the the close method closes the key vault client.  For the unified tests, that's ok because the lifetime of the the MongoClient and ClientEncryptions instances are the same, so we can get away with this.

It would probably work to grab the MongoClientSettings from the MongoClient (it will have the appropriate listeners), but it doesn't seem worth the effort.